client/src/main/scala/org/apache/celeborn/client/LifecycleManager.scala

```diff
@@ -1059,7 +1053,7 @@ class LifecycleManager(appId: String, val conf: CelebornConf) extends RpcEndpoin
     }
   }
 
-  private def requestDestroy(endpoint: RpcEndpointRef, message: Destroy): DestroyResponse = {
+  private def requestWorkerDestroy(endpoint: RpcEndpointRef, message: Destroy): DestroyResponse = {
```

Comment:
I think it should be renamed to 'requestWorkerDestroySlots' and the Destroy message should also be renamed.
The name 'WorkerDestroy' is misleading.

Reply:
Year, should do like this. Let me do this first.

Reply:
OK.